src/factory/tests/HeaderPredicateFactory.mock.ts

```diff
@@ -0,0 +1,343 @@
+// (C) 2007-2018 GoodData Corporation
+import { AFM, Execution } from '@gooddata/typings';
+import { IHeaderPredicateContext } from '../../interfaces/HeaderPredicate';
+
+export const afm: AFM.IAfm = {
+    measures: [
+        {
+            alias: 'Measure defined by uri',
+            definition: {
+                measure: {
+                    item: {
+                        uri: '/measureUri.uri'
+                    }
+                }
+            },
+            localIdentifier: 'measureUri.localIdentifier',
+            format: '#,##0.00'
+        },
+        {
+            alias: 'Measure defined by identifier',
+            definition: {
+                measure: {
+                    item: {
+                        identifier: 'measureIdentifier.identifier'
+                    }
+                }
+            },
+            localIdentifier: 'measureIdentifier.localIdentifier',
+            format: '#,##0.00'
+        },
+
+        {
+            alias: 'Ad-hoc uri-based measure',
+            definition: {
+                measure: {
+                    aggregation: 'count',
+                    item: {
+                        uri: '/attributeUri.uri'
+                    }
+                }
+            },
+            localIdentifier: 'measureAdhocUri.localIdentifier',
+            format: '#,##0.00'
+        },
+        {
+            alias: 'Ad-hoc identifier-based measure',
+            definition: {
+                measure: {
+                    aggregation: 'count',
+                    item: {
+                        identifier: 'attributeIdentifier.identifier'
+                    }
+                }
+            },
+            localIdentifier: 'measureAdhocIdentifier.localIdentifier',
+            format: '#,##0.00'
+        },
+
+        {
+            alias: 'Compare PP measure from uri-based measure',
+            localIdentifier: 'measureUri.PP.localIdentifier',
+            definition: {
+                previousPeriodMeasure: {
+                    measureIdentifier: 'measureUri.localIdentifier',
```

Comment:
As we discussed in person `uriMeasure` would be easier to understand

Reply:
I would maybe suggest not using the dot as it seems it has some meaning which it hasn't. 
`uriMeasureLocalIdentifier` seems better to me.

Reply:
I have renamed all mocks. Hope it is more expressive and correct.